### PR TITLE
[DOC,FIX] Typo@include_seqan_file

### PR DESCRIPTION
**Pull request by beekalam**

### CODE REVIEW

File: include/seqan/file/string_external.h [hunk 1]
```diff
@@ -94,7 +94,7 @@ namespace seqan
  *
  * @section Remarks
  *
- * When using this configuration ,th eSize type o fthe ExternalString is Size type of <tt>TFile</tt>.  Normally, this is
+ * When using this configuration ,the Size type of the ExternalString is Size type of <tt>TFile</tt>.  Normally, this is
```
Comment (rrahn) [inline include/seqan/file/string_external.h]: highlight Size if referring to meta-function. Like wrap in @link @endlink or in \<tt\> \</tt\>

File: include/seqan/file/string_external.h [hunk 2]
```diff
@@ -127,7 +127,7 @@ namespace seqan
  *            class String<TValue, External<TConfig> >;
  *
  * @tparam TValue  The type that is used for the items/characters stored in the string.
- * @tparam TConfig A structure to confgure the external string.  Defaults to <tt>ExternalConfigLarge&lt;&gt;.  See
+ * @tparam TConfig A structure to configure the external string.  Defaults to <tt>ExternalConfigLarge&lt;&gt;.  See
```
Comment (rrahn) [inline include/seqan/file/string_external.h]: Double whitespace before Defaults

File: include/seqan/file/string_external.h [hunk 3]
```diff
@@ -1146,7 +1146,7 @@ namespace seqan
  * @signature String::String(file);
  * @signature String::String(fileName[, openMode]);
  *
- * @param[in]     file     The @link File @endlink to use for reading and writing.  You must ensture that
+ * @param[in]     file     The @link File @endlink to use for reading and writing.  You must ensure that
```
Comment (rrahn) [inline include/seqan/file/string_external.h]: Double whitespace?

Comment (beekalam) [inline include/seqan/file/string_external.h]: actually Double white spaces were not from my branch. They exist also in the master branch.

**Comment by rrahn** [comment]
@beekalam we have added a contributor license agreement, which every contributor needs to sign before pushing a PR. Thanks a lot for you support.

**Comment by rrahn** [comment]
lgtm!

**Comment by rrahn** [comment]
I think only fixing typos might not affect the CLA much. So I will merge it.

**Pull request merged**